extensions/wikia/CuratedContent/templates/CuratedContentSpecial_index.mustache

```diff
@@ -7,9 +7,9 @@
 	<ul>
 		{{{list}}}
 		{{^list}}
```

Comment:
I talked to @vforge and we agreed not to add item to featured because featured is optional